acquire/acquire.py

```diff
@@ -134,12 +134,13 @@ def misc_unix_user_homes(target: Target) -> Iterator[fsutil.TargetPath]:
     yield from itertools.chain.from_iterable(home_dirs)
 
 
-def misc_osx_user_homes(target: Target) -> Iterator[fsutil.TargetPath]:
+def misc_macos_user_homes(target: Target) -> Iterator[fsutil.TargetPath]:
     yield from itertools.chain(target.fs.path("/Users/").glob("*"), misc_unix_user_homes(target))
 
 
 MISC_MAPPING = {
-    "osx": misc_osx_user_homes,
+    "macos": misc_macos_user_homes,
+    "osx": misc_macos_user_homes,  # legacy
```

Comment:
You don't add a legacy mapping anywhere else, might as well remove this one too?